src/util/get-files.ts

```diff
@@ -92,7 +114,7 @@ const clearRelative = function(str) {
  * @return {String} results or `''`
  */
 
-const maybeRead = async function(path, default_ = '') {
+const maybeRead = async function<T>(path: string, default_: T) {
   try {
     return await fs.promises.readFile(path, 'utf8');
   } catch (err) {
```

Comment:
Originally, default was set to the empty string. Instead, I make the calling function pass in a default value so that we can utilize generics. This makes typescript understand if the value will be nullable or not.